src/components/SQForm/SQForm.tsx

```diff
@@ -1,19 +1,45 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import Grid from '@material-ui/core/Grid';
+import {GridProps} from '@material-ui/core';
 import {Formik, Form} from 'formik';
 import {useDebouncedCallback} from 'use-debounce';
 import * as Yup from 'yup';
 import {useInitialRequiredErrors} from '../../hooks/useInitialRequiredErrors';
 
+interface SQFormProps {
+  /** Form Input(s) */
+  children: React.ReactNode;
+  /** Bool to pass through to Formik. https://formik.org/docs/api/formik#enablereinitialize-boolean */
+  enableReinitialize?: boolean;
+  /** Form Entity Object */
+  initialValues: Record<string, unknown>;
+  /** Any prop from https://material-ui.com/api/grid */
+  muiGridProps?: GridProps;
+  /**
+   * Form Submission Handler | @typedef onSubmit: (values: Values, formikBag: FormikBag) => void | Promise<any>
+   * IMPORTANT: If onSubmit is async, then Formik will automatically set isSubmitting to false on your behalf once it has resolved.
+   * This means you do NOT need to call formikBag.setSubmitting(false) manually.
+   * However, if your onSubmit function is synchronous, then you need to call setSubmitting(false) on your own.
+   *
+   * https://jaredpalmer.com/formik/docs/api/withFormik#handlesubmit-values-values-formikbag-formikbag--void--promiseany
+   * */
+  onSubmit: (event: React.SyntheticEvent) => void;
+  /**
+   * Yup validation schema shape
+   * https://jaredpalmer.com/formik/docs/guides/validation#validationschema
+   * */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  validationSchema?: Record<string, any>;
```

Comment:
Can this be `Record<string, unknown>` instead of using `any` ?

Reply:
> Can this be `Record<string, unknown>` instead of using `any` ?

Actually I tried this one, but it was yelling
![image](https://user-images.githubusercontent.com/16385157/127698194-3327224b-fc4b-4e6d-a158-26e7ea781cb8.png)

Reply:
@AamirMaan It looks like `Yup` has a type for schemas. If you look in `SQForm.stories.js` line 217 and mouse over `validationSchema` you can see what the schemas look like. But in this we don't know exactly what schemas we are getting so instead of `any` you should be able to use `Yup.MixedSchema` instead.